smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ServiceGenerator.kt

```diff
@@ -68,6 +73,42 @@ class ServiceGenerator(
                 outputShapeName
             )
         }
+
+        /**
+         * Helper method for generating in-line documentation for operation
+         */
+        private fun renderOperationDoc(model: Model, service: ServiceShape, op: OperationShape, writer: SwiftWriter) {
+            writer.writeShapeDocs(op)
+            writer.writeAvailableAttribute(model, op)
+
+            writer.writeSingleLineDocs { write("") }
```

Comment:
Instead of writing `writer.writeSingleLineDocs { write("") }` we could take advantage of Kotlin's Local Functions. In general it is not good to overuse these but I think this is a case where it makes sense to improve readability.

Inside of this function we can declare a local function inside of renderOperationDoc 
```
fun writeEmptyLine() {
    writer.writeSingleLineDocs { write("") }
}
```

And now within the scope of this function we can write `writeEmptyLine()` instead of `writer.writeSingleLineDocs { write("") }` at all occurrences.

https://kotlinlang.org/docs/functions.html#local-functions